examples/playground/src/styles/verna.scss

```diff
@@ -0,0 +1,40 @@
+.verna-wrapper {
```

Comment:
Please order properties alphabetically. I try to enforce this convention for stylesheets in order to improve readability, homogeneity and prevent doublons.

Reply:
Very good point, specially the avoid doublons :eyes: !